code/go/0chain.net/blobbercore/datastore/postgres.go

```diff
@@ -26,6 +28,7 @@ func (store *postgresStore) Open() error {
 		config.Configuration.DBPassword)), &gorm.Config{
 		SkipDefaultTransaction: true, // https://gorm.io/docs/performance.html#Disable-Default-Transaction
 		PrepareStmt:            true, //https://gorm.io/docs/performance.html#Caches-Prepared-Statement
+		DisableAutomaticPing:   false,
```

Comment:
we don't have to set it to `false` explicitly if it's false by default

Reply:
fixed